src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj

```diff
@@ -32,7 +32,7 @@
 
   <ItemGroup>
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsPkgVer)" Condition="'$(TargetFramework)' != 'netstandard2.1'" />
-    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonPkgVer)" Condition="'$(TargetFramework)' != 'net6.0'" />
+    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonPkgVer)" />
```

Comment:
what triggered this change?

Reply:
I searched the usages of System.Text.Json in the current code base and found the condition is not necessary.
We only used `System.Text.Json.JsonSerializer.Serialize` in `JaegerTagTransformer.cs`.
What's the purpose of adding this package reference only when the framework is not `net6.0` before the change?

Reply:
`net6.0` automatically brings `System.Text.Json` library. We should put the check back.

Reply:
I forget. You are right.